source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -670,6 +675,7 @@ export class CardDesigner extends Designer.DesignContext {
             this._hostContainerChoicePicker = new ToolbarChoicePicker(CardDesigner.ToolbarCommands.HostAppPicker);
             this._hostContainerChoicePicker.separator = true;
             this._hostContainerChoicePicker.label = "Select host app:"
+            this._hostContainerChoicePicker.width = 350;
```

Comment:
looks like an incorrect merge (here and below)?

Reply:
Indeed. Not sure where those came from. Fixed.